winui/Cartesian-Charts/Bubble.md

```diff
@@ -62,102 +62,4 @@ chart.Series.Add(series);
-
-{% tabs %}
-
-{% highlight xaml %}
-
-<chart:SfCartesianChart>
-
-    <chart:SfCartesianChart.PrimaryAxis>
-        <chart:CategoryAxis />
-    </chart:SfCartesianChart.PrimaryAxis>
-
-    <chart:SfCartesianChart.SecondaryAxis>
-        <chart:NumericalAxis />
-    </chart:SfCartesianChart.SecondaryAxis> 
-
-    <chart:BubbleSeries ItemsSource="{Binding Data}" XBindingPath="XValue" YBindingPath="YValue" Size="Size" ShowZeroBubbles="True"/>
-    ...
-<chart:SfCartesianChart>
-
-{% endhighlight %}
-
-{% highlight c# %}
-
-SfCartesianChart chart = new SfCartesianChart();
-CategoryAxis primaryAxis = new CategoryAxis();
-chart.PrimaryAxis = primaryAxis;
-NumericalAxis secondaryAxis = new NumericalAxis();
-chart.SecondaryAxis = secondaryAxis;
-. . .
-BubbleSeries series = new BubbleSeries()
-{
-    ItemsSource = new ViewModel().Data,
-    XBindingPath = "XValue",
-    YBindingPath = "YValue",
-    Size = "Size",
-    ShowZeroBubbles = true,
-};
-
-chart.Series.Add(series);
-...
-
-{% endhighlight %}
-
-{% endtabs %}
-
-![Show zero size bubbles support in WinUI Chart](Bubble_Images/WinUI_bubble_chart_showzerobubble_true.png)
-
-{% tabs %}
-
-{% highlight xaml %}
-
-<chart:SfCartesianChart>
-    
-    <chart:SfCartesianChart.PrimaryAxis>
-        <chart:CategoryAxis />
-    </chart:SfCartesianChart.PrimaryAxis>
-
-    <chart:SfCartesianChart.SecondaryAxis>
-        <chart:NumericalAxis />
-    </chart:SfCartesianChart.SecondaryAxis> 
-    
-    <chart:BubbleSeries ItemsSource="{Binding Data}" XBindingPath="XValue" 
-                        YBindingPath="YValue" Size="Size"
-                        ShowZeroBubbles="False"/>
-    ...
-<chart:SfCartesianChart>
-
-{% endhighlight %}
-
-{% highlight c# %}
-
-SfCartesianChart chart = new SfCartesianChart();
-CategoryAxis primaryAxis = new CategoryAxis();
-chart.PrimaryAxis = primaryAxis;
-NumericalAxis secondaryAxis = new NumericalAxis();
-chart.SecondaryAxis = secondaryAxis;
-. . .
-BubbleSeries series = new BubbleSeries()
-{
-    ItemsSource = new ViewModel().Data,
-    XBindingPath = "XValue",
-    YBindingPath = "YValue",
-    Size = "Size",
-    ShowZeroBubbles = false,
-};
-
-chart.Series.Add(series);
-...
-
-{% endhighlight %}
-
-{% endtabs %}
-
-![ShowZeroBubbles support in WinUI Chart](Bubble_Images/WinUI_bubble_chart_showzerobubble_false.png)
-
 N> You can refer to our [WinUI Bubble Chart](https://www.syncfusion.com/winui-controls/charts/winui-bubble-chart) feature tour page for its groundbreaking feature representations. You can also explore our [WinUI Bubble Chart example](https://github.com/syncfusion/winui-demos/blob/master/chart/Views/Basic%20Charts/BubbleChart.xaml) that shows how to easily configure with built-in support for creating stunning visual effects.
```

Comment:
N> Refer to our [WinUI Bubble Chart](https://www.syncfusion.com/winui-controls/charts/winui-bubble-chart) feature tour page for its groundbreaking feature representations. You can also explore our [WinUI Bubble Chart example](https://github.com/syncfusion/winui-demos/blob/master/chart/Views/Basic%20Charts/BubbleChart.xaml) that shows how to easily configure with built-in support for creating stunning visual effects.